lib/Round.js

```diff
@@ -63,15 +63,7 @@ class Round {
     }
   }
 
-  displayClue(categoryId, pointValue) {
-    return this.clues.find(clue => {
-      return (clue.pointValue === pointValue) && (clue.categoryId === categoryId);
-    }).question;
-  }
```

Comment:
Are you planning to move this to the Clue class as well? I see it's deleted here but not present in the Clue class yet.